newIDE/app/resources/examples/bbtext-object-demo/manipulate-bbtext-object.json

```diff
@@ -0,0 +1,1149 @@
+{
```

Comment:
This file should be named "bbtext-object-demo.json" to work :) I'll fix that.

Reply:
Good job on the example by the way, it's showing a lot of use case and is a great way to check that the object is working properly 👍